nx_parallel/algorithms/centrality/betweenness.py

```diff
@@ -41,28 +41,32 @@ def betweenness_centrality(
     if hasattr(G, "graph_object"):
         G = G.graph_object
```

Comment:
remove this

Reply:
So this is actually present on `main`, and if we remove this here, the tests fails